OutOfSchool/OutOfSchool.WebApi/Controllers/ThumbnailTestController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+using OutOfSchool.BusinessLogic.Services.ThumbnailProcessor;
+using OutOfSchool.WebApi.Enums;
+
+namespace OutOfSchool.WebApi.Controllers;
+
+[ApiController]
+[FeatureGate(nameof(Feature.Images))]
+[AspApiVersion(2)]
+[Route("api/v{version:apiVersion}/[controller]/[action]")]
+public class ThumbnailTestController : ControllerBase
```

Comment:
It does not really need a controller. Hosting and serving of both original and small images would be done by external storage service